src/components/CaptureMatching/CandidateImages.js

```diff
@@ -239,9 +239,9 @@ function CandidateImages({ capture, candidateImgData, sameTreeHandler }) {
                             <Box className={classes.captureInfoDetail}>
                               <AccessTimeIcon />
                               <Typography variant="body1">
-                                {getDateStringLocale(
-                                  candidateCapture.created_at
-                                )}
+                                {(candidateCapture.captured_at &&
+                                  candidateCapture.captured_at.slice(0, 10)) ||
```

Comment:
We should try to be consistent in the date format we use. We're using the locale format elsewhere in the same tool, so it's odd to have ISO format here.